x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/inference/deployment/DeploymentManager.java

```diff
@@ -144,6 +144,7 @@ private void doStartDeployment(TrainedModelDeploymentTask task, ActionListener<T
         });
 
         ActionListener<Boolean> modelLoadedListener = ActionListener.wrap(success -> {
+            logger.debug("[{}] model loaded. Starting result processor thread", task.getModelId());
```

Comment:
It might be worth _also_ putting a debug message at the top of `PyTorchResultProcessor.process()`. Then the timestamp difference between this message and that one will show how long it took for the other thread to get going.